code/src/components/Input1.js

```diff
@@ -0,0 +1,25 @@
+import React from 'react'
+
+export const Input1 = ({setActivity, activity}) => {
+ 
+  const onActivityChange =(event) => {
+    setActivity(event.target.value)
+  }
+  
+  return (
+    <div>
+      <h3>
+        Favourite vacation activity 🏄
+      </h3>
+      <label>
+        Type:
+        <input 
+          type="text"
+          onChange={(event) => {onActivityChange(event)}}
```

Comment:
To simplify the code you should be able to only write onChange={onActivityChange} - since event is the default parameter so you shouldn't need to specify this.